app/models/concerns/hyrax/admin_set_behavior.rb

```diff
@@ -32,6 +32,19 @@ def self.default_set?(id)
         id == DEFAULT_ID
       end
 
+      # Creates the default AdminSet and an associated PermissionTemplate with workflow
+      # rubocop:disable Lint/HandleExceptions
+      def self.create_default!
+        return if AdminSet.exists?(DEFAULT_ID)
+        AdminSet.create!(id: DEFAULT_ID, title: ['Default Admin Set']) do |_as|
+          PermissionTemplate.create!(admin_set_id: DEFAULT_ID)
```

Comment:
Possibly. That's a change in the last few days since this started and maybe reflected in the generic_work_actor_spec: `let(:admin_set) { create(:admin_set, with_permission_template: { with_active_workflow: true }) }`

I can look at adding the workflow bit here.